lookout/style/format/visualizer/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
+    }
+  ],
+  "start_url": "./index.html",
+  "display": "standalone",
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
+}
```

Comment:
@vmarkovtsev that would be possible depending on what we expect out of it; What exactly?

Because I'm wondering given this need if as an alternative it's not better to have this designed by product and built by apps, like the code annotation tool.

Reply:
As discussed IRL, no need for Apps this time since it was enough Hugo coded it himself. It still lacks in usability and interface, thus we'll schedule some time from Ricardo early next to help surfacing issues with a quick interview and implementing a better interface.